frontend-react/e2e/spec/all/managing-your-connection-page.spec.ts

```diff
@@ -0,0 +1,125 @@
+import { scrollToFooter, scrollToTop } from "../../helpers/utils";
+import { ManagingYourConnectionPage } from "../../pages/managing-your-connection";
+import { test as baseTest, expect } from "../../test";
+
+const cards = [
+    {
+        name: "For healthcare organizations",
+        links: [
+            "Manage your public key",
+            "View your submission history",
+            "Login",
+            "contact us",
+        ],
+    },
+    {
+        name: "For public health agencies",
+        links: [
+            "Refer healthcare organizations",
+            "View your dashboard",
+            "Login",
+            "contact us",
+        ],
+    },
+];
+
+export interface ManagingYourConnectionPageFixtures {
+    managingYourConnectionPage: ManagingYourConnectionPage;
+}
+
+const test = baseTest.extend<ManagingYourConnectionPageFixtures>({
+    managingYourConnectionPage: async (
+        {
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        },
+        use,
+    ) => {
+        const page = new ManagingYourConnectionPage({
+            page: _page,
+            isMockDisabled,
+            adminLogin,
+            senderLogin,
+            receiverLogin,
+            storageState,
+            isFrontendWarningsLog,
+            frontendWarningsLogPath,
+        });
+        await page.goto();
+        await use(page);
+    },
+});
+
+test.describe(
+    "Managing Your Connection page",
+    {
+        tag: "@smoke",
```

Comment:
what factors are we using to determine what should be a smoke test or not?

Reply:
We all(devs) decided as a team on 7/3/2024 that we would test all `happy path scenarios` in public facing pages and all `happy path scenarios` for admins in auth pages,

Reply:
awesome! might be worth adding that to the README for any future public facing pages + devs.